app/tests/DB_connection_Tests.cpp

```diff
@@ -1,78 +0,0 @@
-#include <catch2/catch.hpp>
-#include <memory>
-#include <pqxx/pqxx>
-#include <array>
-#include <algorithm>
-
-#include "../src/DB_connection.hpp"
-#include "../src/SQL_Query.hpp"
-
-
-
-std::array<char*,5> inputs = {"I am the exe name","postgresql","5432","sean","Fit"};
-
-//stupid fixture
-No_Pass strategy;
-static DB_connection conn(inputs, &strategy);
-
-TEST_CASE("Does Connection work?", "[DB_connection]"){
-    SQL_Query q("SELECT 1;");
-    conn.execute(&q);
-    int x = q.result.at(0)[0].as<int>();
-    REQUIRE(x == 1);
-}
-
-TEST_CASE("Can Connection read?", "[DB_connection]"){
-    SQL_Query q("SELECT * FROM pg_catalog.pg_tables");
-    conn.execute(&q);
-
-    REQUIRE(strcmp(q.result.column_name(0),"schemaname") == 0);
-    REQUIRE(strcmp(q.result.column_name(1),"tablename") == 0);
-
-}
-
-
-//Purpose of this class is to call destructor on test exit
-class TestTable{
-    public:
-        TestTable(){
-            SQL_Query q("CREATE TABLE test( id int, name varchar(25), date Date);");
-            conn.execute(&q);
-        }
-        ~TestTable(){
-            SQL_Query q("DROP TABLE test");
-            conn.execute(&q);
-        }
-};
```

Comment:
cant this... cause issues? i think you need a dropifexists before your create table test. imagine your code broke or sql connection broke halfway through this method, this test would never pass again